sys/posix/netdb/netdb.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright(C) 2017 Imagination Technologies Limited and/or its
+ *              affiliated group companies.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+/**
+ * @{
+ *
+ * @file
+ *
+ * @author  Francois Berder <francois.berder@imgtec.com>
+ */
+
+#include <netdb.h>
+#include <sys/cdefs.h>
+#include "net/sock/dns.h"
+#include "net/sock/util.h"
+
+#ifndef POSIX_DNS_SERVER
+#error "POSIX_DNS_SERVER must be set"
+#endif
+
+#define str(s) #s
```

Comment:
Why not require the define being a string with quotes? We do that in other places.
Also IMO we should handle setting DNS servers centrally, independent from POSIX.